### ISSUE TITLE
Request: Add a spread query

### ISSUE THREAD

**Issue by virus-zip** [orionprotocol/trading-sdk#30]
Instead of calculating the spread difference on your own, a simple query could be added to retrieve the number

**Comment by eugene-mac** [comment]
> Instead of calculating the spread difference on your own, a simple query could be added to retrieve the number

@virus-zip Did you mean adding subscriber method to WS which will subscribe for orderbook spread?

**Comment by virus-zip** [comment]
No, just for a single orderbook. In order to not calculate the difference yourself, you could just use something like orderbook.currentSpread() to return the current spread as a number or string.
